### ISSUE TITLE
[Critical] Button: Button does not have a name

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#590]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87716 |
| **Checkpoint** | 4.1 Compatible › Name, Role, Value (4.1.2.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/11/26 at 10:58 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The button is missing an accessible name/label. As a result its functionality or purpose is not conveyed to screen reader users.
In Settings menu, select BagIt Profiles, then New, then accept the default of AP Trust, then select Create. Expand the Tag Files, select Add New Tag File.  In the resulting, New Tag File dialog, the close button is not labeled. See Settings User Experience video at 11:55. 
Also occurs if you select one of the tags to edit (Tag Definition dialog), see Settings User Experience video at 13:20.

## Recommendation

Rule : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

How To Fix: 
Fix this issue by using any of the following techniques:
1. Use the innertext of the <button> or role="button" element

<button>Apply now!</button>

<button class="apply-btn"><span class="sr-text">Apply now!</span></button>

2. Use a non-empty aria-label attribute on the <button> or role="button" element.

<button class="apply-btn" aria-label="Apply now!"></button>

<span role="button" class="apply-btn" aria-label="Apply Now!"></span>

Reference: 
W3C ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/#aria-label

Background: 
Buttons must have discernible text that describes the button's function or action for screen reader and other assistive technology users.

## Screenshots

![01_button label1](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87716/01_button_label1.png)
![01_button label](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87716/01_button_label.png)
